### ISSUE TITLE
error testcase

### ISSUE THREAD

**Issue by drzraf** [sflynn1812/xslt-diff#3]
### `a.xml` == `b.xml`:
```xml
<item xmlns:wp="http://foo/bar/">
  <a/><a/><a/><a/><a/><a/><a/><a/><a/><a/><a/><a/><a/><a/><a/><a/><a/><a/><a/>
  <wp:b><a/></wp:b>
  <a/><a/>
  <wp:b><key/></wp:b>
</item>
```

### `Output`:
```xml
<?xml version="1.0" encoding="utf-8"?>
<root xmlns:msxsl="urn:schemas-microsoft-com:xslt">
  <compare-result>
    <mismatch>
      <key xmlns:wp="http://foo/bar/"/>
    </mismatch>
  </compare-result>
</root>
```

### `Expected`:
```xml
<?xml version="1.0" encoding="utf-8"?>
<root xmlns:msxsl="urn:schemas-microsoft-com:xslt">
  <compare-result/>
</root>
```

### `$ xsltproc -V`
```
Using libxml 20904, libxslt 10129 and libexslt 817
xsltproc was compiled against libxml 20904, libxslt 10129 and libexslt 817
libxslt 10129 was compiled against libxml 20904
libexslt 817 was compiled against libxml 20904
```

**Comment by sflynn1812** [comment]
Hi - sorry I had converted a pre-release copy of the stylesheet that did not work. Version 1.2 gives proper output.

**Comment by sflynn1812** [comment]
Just a general FYI, the bad version was an attempt at mitigating stackoverflow problems when the xml become in the thousands.  Earlier versions worked correctly, which I have corrected and posted in version 1.2.  I appreciate the testing, and please feel free to post any additional bugs you may find.
